data/json/items/generic/bathroom_house.json

```
@@ -0,0 +1,233 @@
+[
+  {
+    "id": "base_toiletries",
+    "name": "generic grooming",
+    "type": "GENERIC",
+    "description": "generic item template",
+    "symbol": "+",
+    "color": "blue",
+    "price": 200,
+    "price_postapoc": 0,
+    "weight": 100,
+    "volume": "50 ml",
+    "material": "plastic",
+    "bashing": 1,
+    "to_hit": -1
+  },
+  {
+    "id": "soap_holder",
+    "name": "soap dish",
+    "name_plural": "soap_dishes",
```

```suggestion
    "name": { "str": "soap dish", "str_pl": "soap dishes" },
```
Please replace name and name_plural with this format